README.md

```diff
@@ -43,6 +43,17 @@ outgoing message.
 *   `Skip rows` - if you know that the incoming CSV file has certain number of headers you can indicate to skip them. The supported values are `None`, `First row`, `First two`, `First three` and `First four`.
 *   `Data columns` - here the values will be added dynamically based on the values in the `CSV Header` field. Here each data column will be listed with the name, Data Type and the Format to enable further configuration.
 
+### Read CSV file from URL v2
```

Comment:
I don't think it makes sense to have the read action also exist as a trigger. I would be in favor of removing it (and depricating the existing trigger.)

Reply:
Trigger removed, old read deprecated